src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -107,7 +107,7 @@ def get_organizations(
         *,
         user_id: int,
         only_visible: bool = False,
-    ) -> List[RpcOrganizationSummary]:
+    ) -> List[RpcOrganizationMapping]:
```

Comment:
This type was wrong before, I don't yet understand how mypy was ok with the code before.